tasks/main.yml

```diff
@@ -41,10 +41,15 @@
   tags:
     - designate-install
 
+- include: designate_db_setup.yml
+  when: inventory_hostname == groups['designate_all'][0]
+  tags:
+    - designate-install
+
 - name: Flush handlers
   meta: flush_handlers
 
-- include: designate_db_setup.yml
+- include: designate_pool_setup.yml
```

Comment:
Is the db_sync not being done at all anymore?   Should the `tasks/designate_db_setup.yml` file be removed?

Reply:
It's not being removed, it moved up in the file, but it is still being called.

Reply:
oh, so it is